src/persist/src/nemesis/generator.rs

```diff
@@ -200,6 +200,7 @@ impl ReqGenerator {
             .copied()
             .unwrap_or_default();
         let ts = stream_sealed_ts + rng.gen_range(0..5);
+        state.seal_frontier.insert(stream.clone(), ts);
```

Comment:
its a little hard for me to wrap my head around this one because the command might fail if the storage is unavailable or the runtime is down right - should we still update the seal frontier in that case?

my understanding is that updating the seal frontier does no harm, it just bumps up the time at which we'll issue future seal requests.

Also, I realized that the since frontier has a similar bug where afaict we never write to the hashtable containing sinces